README.md

```diff
@@ -148,15 +148,11 @@ A detailed post on Git commit messages: [How To Write a Git Commit Message](http
 
 ### Internationalization
 
-We have [Transifex](https://www.transifex.com/oasisprotocol/oasis-wallet-web/) to easily contribute translations.
+Translating: We have [Transifex](https://www.transifex.com/oasisprotocol/oasis-wallet-web/) to easily contribute translations.
 
-Oasis-wallet uses [react-i18next](https://react.i18next.com/) for I18n. You can simply use the [useTranslation hook](https://react.i18next.com/latest/usetranslation-hook) inside your components to add additional i18n-ready strings. You can then export the new keys to the translation files by running
+Development: Oasis-wallet uses [react-i18next](https://react.i18next.com/) for I18n. You can simply use the [useTranslation hook](https://react.i18next.com/latest/usetranslation-hook) inside your components to add additional i18n-ready strings. You can then export the new keys to the [English translation.json](src/locales/en/translation.json) by running `yarn run extract-messages`.
```

Comment:
Can you leave

```
yarn run extract-messages
```
on a separate line?

Reply:
I don't like it as much, because it breaks the categories `Translating`, `Development`, `Update from Transifex`. Even if someone is `Translating`, this unrelated command will stand out to them. Making it a list helps a bit, but not much

| Inline | Own line | Alternative: list |
| --- | --- | --- |
| ![1](https://user-images.githubusercontent.com/3758846/172747983-8f3ae491-a7a9-4fee-b048-04a7acc4fa70.png)  |  ![2](https://user-images.githubusercontent.com/3758846/172747988-f5dce9c3-4467-49ab-b181-a2200d674a17.png) |  ![3](https://user-images.githubusercontent.com/3758846/172747991-b0679ce3-7e9f-43e9-bfbd-3dd40cdeab54.png) |